.ci/azure/windows.yml

```diff
@@ -114,7 +117,24 @@ jobs:
     continueOnError: false
 
   - script: |
-      set PATH=$(TEST_ENV_PATH)
+      powershell -command "Invoke-WebRequest  -Uri  https://d30ikxcvcet9xo.cloudfront.net/repositories/openvino/ci_dependencies/test/samples_smoke_tests_data.zip -OutFile  $(REPO_DIR)\tests\samples_smoke_tests\samples_smoke_tests_data.zip"
+      powershell -command "Expand-Archive -Force -Path $(REPO_DIR)\tests\samples_smoke_tests\samples_smoke_tests_data.zip $(REPO_DIR)\tests\samples_smoke_tests"
+      python -m pip install  --upgrade pip
+      python -m pip install -r $(REPO_DIR)\tests\samples_smoke_tests\requirements.txt
+      set IE_APP_PATH=$(BIN_DIR)
+      set IE_APP_PYTHON_PATH=$(REPO_DIR)\inference-engine\ie_bridges\python\sample\
+      set PYTHONPATH=$(REPO_DIR)\model-optimizer\;$(REPO_DIR)\tests\samples_smoke_tests\;$(BIN_DIR)\$(BUILD_TYPE)\python_api\python3.7\;$(REPO_DIR)\ngraph\python\src\
+      set PATH=$(REPO_DIR)\inference-engine\temp\tbb\bin;$(REPO_DIR)\inference-engine\temp\opencv_4.5.1\opencv\bin;$(BIN_DIR);$(BIN_DIR)\$(BUILD_TYPE);$(REPO_DIR)\inference-engine\temp\gna_02.00.00.1047.1\win64\x64;%PATH%
+      set SHARE=$(REPO_DIR)\tests\samples_smoke_tests\samples_smoke_tests_data\
+      set WORKSPACE=$(REPO_DIR)
+      set IE_APP_PYTHON_TOOL_PATH=$(REPO_DIR)\inference-engine\tools\
+      python -m pytest $(REPO_DIR)\tests\samples_smoke_tests\  --env_conf $(REPO_DIR)\tests\samples_smoke_tests\env_config.yml -s --junitxml=TEST-SamplesSmokeTests.xml
+    workingDirectory: $(REPO_DIR)
+    displayName: 'SamplesSmokeTests'
+    continueOnError: false
+
+  - script: |
+      set PATH=$(REPO_DIR)\inference-engine\temp\tbb\bin;%PATH%
```

Comment:
Wrong merge
must be set PATH=$(TEST_ENV_PATH

Reply:
Done